components/data-feed-service/Makefile

```diff
@@ -1,13 +1,10 @@
-GOFILES=$(shell go list ./... | grep -v '/vendor|asset\.go|proto/' | sed 's/^_//')
-GOPATH=$(shell go env GOPATH)
-
 include ../../Makefile.common_go
 
 default: ci
 
 ci: review build
 
-review: lint vet
```

Comment:
There are a few leftover `vet` targets, all in `.PHONY` lines:
components/authn-service/Makefile
components/data-feed-service/Makefile
components/local-user-service/Makefile
components/session-service/Makefile